vendor/github.com/dcos/dcos-go/dcos/constants.go

```diff
@@ -20,7 +20,8 @@ const (
 func GetFileDetectIPLocation() string {
 	switch runtime.GOOS {
 	case "windows":
-		return "/opt/mesosphere/bin/detect_ip.ps1"
+        return "c:\\dcos\\opt\\bin\\detect_ip.ps1"
```

Comment:
Can you update dcos-go to contain this update https://github.com/dcos/dcos-diagnostics/blob/5856695d58b817e07f590345abc813a56e3a9b2c/go.mod#L10

Reply:
https://github.com/dcos/dcos-go/blob/3b86d9c7fac36be9d16fc1d22e5a5dc54738aac0/dcos/constants.go#L23